### PR TITLE
Not toggle floating on a CT_FLOATING_CON

### PR DESCRIPTION
**Pull request by hwangcc23**
See the issue #1875.
Forbit the command to toggle floating on a CT_FLOATING_CON to avoid the crash.

### CODE REVIEW

File: src/floating.c [hunk 1]
```diff
@@ -369,6 +369,12 @@ void floating_disable(Con *con, bool automatic) {
  *
  */
 void toggle_floating_mode(Con *con, bool automatic) {
+    /* forbit the command to toggle floating on a CT_FLOATING_CON */
```
Comment (Airblader) [inline src/floating.c]: s/forbit/forbid/

File: src/floating.c [hunk 2]
```diff
@@ -369,6 +369,12 @@ void floating_disable(Con *con, bool automatic) {
...
+    if (con->type == CT_FLOATING_CON) {
+        LOG("not toggle floating on a CT_FLOATING_CON\n");
```
Comment (Airblader) [inline src/floating.c]: You're probably not a native English speaker, so no worries, but can we rephrase this to 

    ELOG("Cannot toggle floating mode on %p because it is of type CT_FLOATING_CON.\n", con);

File: src/floating.c (hunk 1 above)
Comment (hwangcc23) [inline src/floating.c]: Sorry for the typo.

File: src/floating.c (hunk 2 above)
Comment (hwangcc23) [inline src/floating.c]: Thanks. Done.  :)
